lib/tasks/data.rake

```diff
@@ -79,3 +79,37 @@ namespace :google_drive do
     puts "#{documents_deleted} Documents deleted. Have a good day!"
   end
 end
+
+namespace :vacancies do
+  desc "Fix about_school fields of live vacancies"
+  task fix_about_school: :environment do
+    fix_before = Date.new(2021, 5, 14)
+
+    organisation_ids = Organisation.not_closed.where.not(description: [nil, ""]).pluck(:id)
+
+    vacancies = Vacancy.live
+                       .distinct
+                       .joins(:organisation_vacancies)
+                       .where(organisation_vacancies: { organisation_id: organisation_ids })
+                       .where("vacancies.created_at < ?", fix_before)
+
+    puts "Found #{vacancies.count} vacancies created before #{fix_before} that may have a wrong about_school field"
+
+    puts "Get ready to fix them!"
+
+    fixed_vacancies = 0
+
+    vacancies.find_each do |vacancy|
+      organisation_description = vacancy.organisations.map(&:description).compact.first
```

Comment:
You could use `parent_organisation` to make sure that you get the right organisation

Reply:
For example, a job at a school within a trust should have a description from the school, rather than the trust

Reply:
How do we determine which organisation description to show on the about_school pre-populated field?

Reply:
it is not the parent organisation that pre-populates that field, but the organisation that the user selects when creating the vacancy. We cannot get that when we are running the script, so this will get the first description that is populated.

Reply:
If we really wanted to, could we check if any of the organisation descriptions match?

Reply:
yes we can, but in my opinion it is not worth it to overcomplicate this script, the chances to fix an extra description are very low. We have only 15 live vacancies with multiple organisations

Reply:
I still think the logic is wrong: if the vacancy is at multiple schools, the first description that is populated might be one of those schools, rather than the first parent organisation. This is what populates the field:

```ruby
def vacancy_or_organisation_description(vacancy)
  vacancy.about_school.presence || vacancy.parent_organisation.description.presence
end
```

Reply:
Fixed it